examples/config-scrub.json

```diff
@@ -0,0 +1,26 @@
+{
+    "distSpecVersion":"1.0.1",
+    "storage": {
+        "rootDirectory": "/tmp/zot",
+        "scrub": {
+            "enable": true,
```

Comment:
Do we need the "enable" here? just having scrub and its interval should suffice imo

Reply:
Done! Now if scrub interval is set then scrub will be enabled. If "scrub" field is  specified but no value provided for "interval" then scrub will not be enabled.